assets/templates/devkit/BOM_DEV_SOFTWARE.md

```diff
@@ -1,18 +1,27 @@
 | Software       | Website                                   |
 |:---------------|:------------------------------------------|
-| bash           | https://www.gnu.org/software/bash/        |
+| bash           | https://www.gnu.org/software/bash         |
+| ctags          | http://ctags.sourceforge.net              |
+| direnv         | https://direnv.net                        |
 | docker         | https://www.docker.com                    |
-| docker-compose | https://docs.docker.com/compose/          |
+| docker-compose | https://docs.docker.com/compose           |
+| fossa-cli      | https://github.com/fossas/fossa-cli       |
```

Comment:
probably need to add fzf

I also install stern for tailing k8s logs and tilt as a front end to skaffold

Reply:
done

Reply:
BOM fixed